mozdownload/scraper.py

```diff
@@ -451,6 +451,8 @@ def get_latest_build_date(self):
                           months.entries[-1] + '/')
             parser = self._create_directory_parser(url)
             parser.entries = parser.filter(r'.*%s' % self.platform_regex)
+            if not parser.entries:
+                raise errors.NotSupportedError('Selected platform not available')
```

Comment:
Given that it is not clear at this point why no build has been found - it could also be something else beside the requested platfrom - lets use a general error message like:

```suggestion
                raise errors.NotSupportedError('No builds have been found')
```